Test_design.md

```diff
@@ -0,0 +1,64 @@
+# Test design for selected functionalities of the website pracuj.pl
+
+## Logowanie

```

Comment:
Podział i testy fajne, ale proponuje podzielić ten test design na mniejsze pliki, które będa grupowały testy w obębie danej funkcjonalności. W przedstawionych poniżej testach widzę 2 główne funkcjonalności:
- logowanie (wraz z odzyskiwaniem hasła)
- wyszukiwanie ofert (wraz z filtrami)
Dlaczego warto to podzielić? Bo same testy też będzie grupował funkcjonalnie w plikach *.py, a więc później możesz nazwać plik *.md z test designem tak samo jak ten *.py i umieścić w tym samym folderze. Będzie je łatwo znaleźć itp.